src/data-access/repositories/mongo/mongo-ingested-entity-converter.ts

```diff
@@ -58,6 +58,7 @@ export interface MongoIngestedSegment extends MongoId {
   metaData?: unknown // This is the current spelling in the database from Core... TOD: Update when we control Ingest
   budgetDuration?: number
   invalidity?: { reason: string }
+  definesShowStyleVariant?: boolean
```

Comment:
I agree with Anders that this should just default to false instead of being optional.

Reply:
Goes for all the other places this is defined as well.

Reply:
It is no longer optional and now defaults to false!